packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -89,6 +90,7 @@ export default function ImpactVectorGraph({ data }: { data: DataSet[] }) {
             }
           }}
         >
+          <Legend />
```

Comment:
I didn't really find a good way to label y-axis directly, so I added a legend to the bottom of the graph like this:

<img width="879" alt="Screenshot 2024-02-24 at 10 36 24 AM" src="https://github.com/BuidlGuidl/impact-calculator/assets/22231097/4ca86f6f-fd5a-4c13-99d2-bb7ec638558b">